### PR TITLE
docs: update cmdlet markdown

### PR DESCRIPTION
**Pull request by tenthirtyam**
### Summary

Updated the markdown formatting for cmdlet descriptions.

### Type

- [ ] Bugfix
- [ ] Enhancement or Feature
- [ ] Code Style or Formatting
- [ ] Documentation
- [x] Refactoring
- [ ] Chore
- [ ] Other
        Please describe:

### Breaking Changes?

- [ ] Yes, there are breaking changes.
- [x] No, there are no breaking changes.

### Test and Documentation

- [x] Tests have been completed.
- [x] Documentation has been added or updated.

### CODE REVIEW

File: docs/documentation/functions/licenses/New-VCFLicenseKey.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Accept wildcard characters: False
 
 ### -productType
 
-Specifies the product type for the license key. One of: SDDC_MANAGER, VCENTER, VSAN, ESXI, NSXT.
+Specifies the product type for the license key. One of: SDDC_MANAGER, VCENTER, VSAN, ESXI, NSXT, NSXV, WCP, HORIZON_VIEW.
```
Comment (LifeOfBrianOC) [inline docs/documentation/functions/licenses/New-VCFLicenseKey.md:47]: Are NSXV & HORIZON_VIEW still valid in 4.x/5/x?

**Review by LifeOfBrianOC — APPROVED** [review]
Looks good bar the one question on NSXV & Horizon

**Comment by tenthirtyam** [comment]
> Are NSXV & HORIZON_VIEW still valid in 4.x/5/x? 


Great catch! Will removed from cmdlet and docs.

File: docs/documentation/functions/licenses/New-VCFLicenseKey.md (hunk 1 above)
Comment (tenthirtyam) [reply docs/documentation/functions/licenses/New-VCFLicenseKey.md]: Resolved in https://github.com/vmware/powershell-module-for-vmware-cloud-foundation/pull/226/commits/f6245c72edb6e807bf420610c77d00c6023c1d65

**Review by LifeOfBrianOC — APPROVED** [review]
Ship it :)

**Pull request merged**